pkg/curatedpackages/bundle.go

```diff
@@ -0,0 +1,117 @@
+package curatedpackages
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/go-logr/logr"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+	"github.com/aws/eks-anywhere-packages/pkg/artifacts"
+	"github.com/aws/eks-anywhere-packages/pkg/bundle"
+	"github.com/aws/eks-anywhere-packages/pkg/testutil"
+	"github.com/aws/eks-anywhere/pkg/constants"
+	"github.com/aws/eks-anywhere/pkg/executables"
+)
+
+const (
+	Cluster         = "cluster"
+	Registry        = "registry"
+	RegistryBaseRef = "public.ecr.aws/q0f6t3x4/eksa-package-bundles"
+)
+
+func GetLatestBundle(ctx context.Context, kubeConfig string, source string, kubeVersion string) (*api.PackageBundle, error) {
+	var (
+		packageBundle *api.PackageBundle
+		err           error
+	)
+
+	switch strings.ToLower(source) {
+	case Cluster:
+		packageBundle, err = getActiveBundleFromCluster(ctx, kubeConfig)
+	case Registry:
+		packageBundle, err = getLatestBundleFromRegistry(ctx, kubeVersion)
+	}
+
+	if err != nil {
+		return nil, err
+	}
+	return packageBundle, nil
+}
+
+func createBundleManager(kubeVersion string) (manager bundle.Manager) {
+	versionSplit := strings.Split(kubeVersion, ".")
+	major, minor := versionSplit[0], versionSplit[1]
+	log := logr.Discard()
```

Comment:
Whats the rationale for the noop logger?

Reply:
There is a need for a logger in the implementation of the NewBundleManager and since I don't need the logger for my specification, there was the need for the noop logger.

Reply:
Generally it should be the main func or some derivative of the main func that stipulates what logger to use. In constructing the logger in-line like this we deprive ourselves of the log output from the bundle manager when we really need it (aka there's a problem). 

I think you'll want to pass the logger as an argument and use it in `NewBundleManaer()`.